src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
+    "sr-acl"
```

Comment:
The acl that's covered by this command is very specific to schema registry. Tomorrow if we have acl for another product then we might be in a fix with the generic name. The only other way I can think of is make acl a command and sr as a sub command and then identify the delegate script based on command and subcommand. This could make the implementation slightly complex